src/progress-bar/progress-bar.test.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import {render} from "@testing-library/react";
+import "@testing-library/jest-dom";
+import {create} from "react-test-renderer";
+import ProgressBar, {ProgressBarProps} from "./ProgressBar";
+import {testA11y} from "../core/utils/test/testUtils";
+
+describe("<ProgressBar />", () => {
+  const defaultProgressBarProps: ProgressBarProps = {
+    testid: "progress-bar",
+    percentage: 57,
+    style: {backgroundColor: "blue", completedColor: "yellow", trackColor: "orange"}
+  };
+
+  it("should render correctly", () => {
+    render(<ProgressBar {...defaultProgressBarProps} />);
+  });

```

Comment:
Do we really need to wrap this with an `it` ?
```suggestion
  render(<ProgressBar {...defaultProgressBarProps} />);
```